### DISCUSSION TITLE
Release time for packages with pytorch 2.x

### DISCUSSION THREAD

**Discussion by ZhiliangWu** [frgfm/torch-cam#221] [General]
Hi all, 

I noticed that there have been several updates to bump up the torch version, such as the commit [0b201aa](https://github.com/frgfm/torch-cam/commit/0b201aa88c15efab962f4da4ee0938af4c69b4db). Is there any plan to release a new package version in a near future?

**Comment by ZhiliangWu** [comment]
@frgfm

**Reply by frgfm** [reply]
Hi @ZhiliangWu :wave:

Thanks for the headsup! Sure, I'll publish a new version today :+1:

**Reply by ZhiliangWu** [reply]
Awesome!
